pydax/_dataset.py

```diff
@@ -161,6 +161,36 @@ def is_downloaded(self) -> bool:
         return self._data_dir.is_dir() and len(os.listdir(self._data_dir)) > 0
 
 
+def handle_name_version_params(func: Callable) -> Callable:
+    """Decorator for handling ``name`` and ``version`` parameters."""
+    def wrapper_decorator(name: str,
+                          version: Optional[str] = None,
```

Comment:
Allowing `version` to be both type `str` and `None` causes mypy errors later on (ln 214, 217, and 248). Any suggestions for a better way to handle here? 

My ideas were to either default `version` to an empty string instead of `None` or to reassign its value to a new variable that is just a string type for mypy purposes in the `load_dataset` and `get_dataset_metadata` functions.

Reply:
mypy produces errors because those 3 lines assume that `version` is a string? We can either skip those errors, or if you think it's too ugly (which is totally avoidable here), how about defaulting to `'latest'`, instead of an empty string?